demo/index.html

```diff
@@ -76,21 +82,21 @@
     rows='[ ["Col1", 5.0],["Col2", 5.0],["Col3", 5.0] ]'>
   </google-chart>
 
-  <script>
-    (function() {
-      function getRandomValue() {
-        return Math.random() * 10;
-      }
+  <script type="module">
+import '@polymer/promise-polyfill/promise-polyfill-lite.js';
+import '../google-chart.js';
+function getRandomValue() {
+  return Math.random() * 10;
+}
 
-      var chart = document.getElementById('mutating-chart');
+var chart = document.getElementById('mutating-chart');
 
-      window.setInterval(function() {
-        chart.rows = [["Col1", getRandomValue()],
-                      ["Col2", getRandomValue()],
-                      ["Col3", getRandomValue()]];
-      }, 3000);
-    }());
-  </script>
+window.setInterval(function() {
+  chart.rows = [["Col1", getRandomValue()],
+                ["Col2", getRandomValue()],
+                ["Col3", getRandomValue()]];
+}, 3000);
+</script>
```

Comment:
indents don't match. intended? Just be consistent throughout.

Reply:
Reformatted.